site/content/guides/monitoring-as-code-cli.md

```diff
@@ -0,0 +1,347 @@
+
+The CLI will determine all the needed changes to be performed to replicate our monitoring code on Checkly. 
+
+```
+MacBook-Air-5:checkly-mac-cli-example hanneslenke$ npx checkly deploy
+? You are about to deploy your project "Website Monitoring" to account 
+"TestCheckly". Do you want to continue? Yes
+Successfully deployed project "Website Monitoring" to account "TestCheckly".
+```
+
+Logging in to our Checkly account, we will see the dashboard has been populated with our three checks, which will soon start executing on their set schedules.
+
+{{< figure src="/guides/images/guides-mac-cli-checks-on-checkly.png" alt="CLI-deployed checks on checkly dashboard" title="CLI-deployed checks on Checkly" >}}
+
+### Monitoring API correctness and performance
+
+Browser checks are now there to keep us informed on the status of our key website flows. What about our APIs, though? Whether they make up the foundation of our service or they are consumed directly by the customer, we need to ensure our endpoints are working as expected. This is easily achieved by setting up API check resources.
+
+{{< tabs "CLI example 3" >}}
+{{< tab "API check resource" >}}
+```ts
+// __check__/api.check.ts
+import { ApiCheck, AssertionBuilder } from '@checkly/cli/constructs'
+
+const api = new ApiCheck('webstore-list-books', {
+  name: 'List Books API',
+  locations: ['eu-central-1','us-west-1'], // overrides the locations property
+  frequency: 1, // overrides the frequency property
+  doubleCheck: true,
+  degradedResponseTime: 5000,
+  maxResponseTime: 10000,
+  request: {
+    method: 'GET',
+    url: 'https://danube-web.shop/api/books',
+    assertions: [
+      AssertionBuilder.statusCode().equals(200),
+      AssertionBuilder.jsonBody('$.length').equals('30')
+    ]
+  }
+})
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+We can now once more run `checkly deploy` to see the new check on Checkly:
+
+{{< figure src="/guides/images/guides-mac-cli-api-checks-on-checkly.png" alt="CLI-deployed api check on checkly" title="Our API and Browser checks on Checkly" >}}
+
+### Alerting
+
+Now that we have our checks in place, we want to set up alerting to ensure we are informed as soon as a failure takes place. Alert channels can be declared as constructs, just like the checks. Let's add the following to our `alert-channels.ts` file:
+
+```ts
+// alert-channels.ts
+import { EmailAlertChannel } from '@checkly/cli/constructs'
+
+const sendDefaults = {
+  sendFailure: true,
+  sendRecovery: true,
+  sendDegraded: false,
+}
+
+export const emailChannel = new EmailAlertChannel('email-channel-1', {
+  address: 'alerts@acme.com',
+  ...sendDefaults
+})
+```
+
+We are setting up things so that we are alerted when our check starts failing, as well as when it recovers. But we still need to decide which checks will subscribe to this channel, and therefore be able to trigger the alerts. This is done by adding the following inside the resource declaration of each check, e.g.:
```

Comment:
```suggestion
We are setting up things so that we are alerted when our Check starts failing, as well as when it recovers. But we still need to decide which Checks will subscribe to this channel, and therefore be able to trigger the alerts. This is done by adding the following inside the resource declaration of each check, e.g.:
```